docs/resources/User.md

```diff
@@ -24,25 +24,26 @@ There are other rules and restrictions not shared here for the sake of spam and
+| username           | string            | the user's username, not unique across the platform                                                  | identify              |
+| discriminator      | string            | the user's Discord-tag                                                                               | identify              |
+| global_name        | ?string           | the user's display name, if it is set. For bots, this is the application name                        | identify              |
+| avatar             | ?string           | the user's [avatar hash](#DOCS_REFERENCE/image-formatting)                                           | identify              |
+| bot?               | boolean           | whether the user belongs to an OAuth2 application                                                    | identify              |
+| system?            | boolean           | whether the user is an Official Discord System user (part of the urgent message system)              | identify              |
+| mfa_enabled?       | boolean           | whether the user has two factor enabled on their account                                             | identify              |
+| banner?            | ?string           | the user's [banner hash](#DOCS_REFERENCE/image-formatting)                                           | identify              |
+| accent_color?      | ?integer          | the user's banner color encoded as an integer representation of hexadecimal color code               | identify              |
+| locale?            | string            | the user's chosen [language option](#DOCS_REFERENCE/locales)                                         | identify              |
+| verified?          | boolean           | whether the email on this account has been verified                                                  | email                 |
+| email?             | ?string           | the user's email                                                                                     | email                 |
+| flags?             | integer           | the [flags](#DOCS_RESOURCES_USER/user-object-user-flags) on a user's account                         | identify              |
+| premium_type?      | integer           | the [type of Nitro subscription](#DOCS_RESOURCES_USER/user-object-premium-types) on a user's account | identify              |
+| public_flags?      | integer           | the public [flags](#DOCS_RESOURCES_USER/user-object-user-flags) on a user's account                  | identify              |
+| avatar_decoration? | ?string           | the user's [avatar decoration hash](#DOCS_REFERENCE/image-formatting)                                | identify              |
+| primary_guild?     | ?UserPrimaryGuild | the user's [primary guild](#DOCS_RESOURCES_USER/user-object-user-primary-guild)                      | identify              |
```

Comment:
our standard format would be making the type column `?[user primary guild](link) object` and not linking to the object in the description - see the `avatar_decoration_data` and `collectibles` objects in https://discord.com/developers/docs/resources/user#user-object-user-structure

looks like those fields don't show in this PR since its old (needs to be updated to fix conflicts)